Documentation/Blazorise.Docs/Components/DocsAttributes.razor.cs

```diff
@@ -1,16 +1,30 @@
 ﻿#region Using directives
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using Blazorise.Docs.Services;
 using Microsoft.AspNetCore.Components;
 #endregion
 
 namespace Blazorise.Docs.Components;
 
-public partial class DocsAttributes
+public partial class DocsAttributes : IDisposable
 {
     #region Methods
 
+    protected override void OnInitialized()
+    {
+        ThemeService.ThemeChanged += OnThemeChanged;
```

Comment:
This is here because we want to make sure the page re-render with the correct theme (`@ThemeContrast`)? 
I think it's enough if the re-render happens in some parent component, resulting in re-render also in here, thus the whole `ThemeService.ThemeChanged`, `StateHasChanged` code has no use in here (and in other child components). It just adds another re-render.

Reply:
It doesn't work all the times if I leave it at parent level. That's why I have the event handler at child components.

Reply:
I see.  In that case I would suggest to investigate the reason why it doesn't work, because it doesn't appear as standard behavior and may backfire badly in the future... 
Also doing the event subscription and un-subscription in every component where you want to use a `ThemeVariable` looks quite painful to me...

My guess is that something is wrong with the non-async version of the ThemeChanged (` async void OnThemeChanged`).. Maybe making it to return Task<string> would solve the issue (really, just a guess).